media/js/firefox/australis/browser-tour.js

```diff
@@ -1120,7 +1179,7 @@ if (typeof Mozilla == 'undefined') {
             } else {
                 localStorage.setItem(this.options.id, 'taken');
             }
-            gaTrack(['_setCustomVar', 5, 'First Time Taking Firefox Tour', firstTime, 2]);
+            return firstTime;
         } catch (e) {}
```

Comment:
Since this function is now returning a value, might be a good idea to return something in the `catch`?